pkg/virtctl/version/version.go

```diff
@@ -40,21 +51,42 @@ type Version struct {
 }
 
 func (v *Version) Run() error {
-	fmt.Printf("Client Version: %s\n", fmt.Sprintf("%#v", version.Get()))
+	fmt.Printf("Client Version: %s\n", fmt.Sprintf("%#v", *clientVersion))
```

Comment:
I would revert back to the old code here and get rid of the `clientVersion` pointer as what you're doing essentially is allocating the `version.Info` structure twice (and making a copy of it) just to save on a `fmt.Sprintf()` call (it's the only field in the `version.Info` which is not defined at compile-time, or maybe it is if the Go compiler optimizes it which is likely).

For the `serverVersion` counterpart it makes sense to cache it as you have to retrieve it from the network but for the client one we already have it statically allocated.

Reply:
Thanks, agree